Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -68,11 +74,23 @@ extension Lambda {
         /// For invocations from the AWS Mobile SDK, data about the client application and device.
         public let clientContext: String?
 
+        // TODO: or should the Lambda "runtime" context and the Baggage context be separate?
+        private var _baggage: BaggageContext
+
+        /// Baggage context.
+        public var baggage: BaggageContext {
```

Comment:
does this need to a `var` vs `let`? is it immutable or would the user ever write on this field?

Reply:
It's currently forced to this via the Carrier's requirement, but it may well be that requirement is quite wrong and should just be `get` rather than `get/set` tbh. We'll look into that. It loops into a few pieces fo feedback @pokryfka had here 👀